website/docs/r/vm_internal_disk.html.markdown

```diff
@@ -0,0 +1,111 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vm_internal_disk"
+sidebar_current: "docs-vcd-vm-internal-disk"
+description: |-
+  Provides a vCloud Director VM internal disk resource. This can be used to create and delete VM internal disks.
+---
+
+# vcd\_vm\_internal\_disk
+
+This can be used to create, update and delete VM internal disks on already created VMs.
+
+~> **Note:** To adjust disk parameters when creating a new VM, please use [override_template_disk](/docs/providers/vcd/r/vapp_vm.html#override-template-disk).
+
+To manage disks which already exist inside a VM, please [import](#importing) them first.
+
```

Comment:
I suggest adding another note here like we have in the override template disk block in VM:
`~> **Note:** Managing disks in VM is possible only when VDC fast provisioned is disabled.`
The idea is to have all crucial exceptions in a single place.

Reply:
Added. Though I feeling uneasy here, cause we talking about vCD functionalities according our experiences. Also here we should talk about terraform things, not vCD. vCD handling is another topic with other KB and documentations. It's my IMHO.